hles/extraction/src/main/scala/org/broadinstitute/monster/dap/HLESurveyExtractionPipeline.scala

```diff
@@ -8,7 +8,7 @@ import Args._
 /** Entry-point for the HLE extraction pipeline. */
 object HLESurveyExtractionPipeline extends ScioApp[Args] {
 
-  val HLESEpoch = "2018-01-01"
+  val HLESEpoch = "2018-01-01 00:00"
```

Comment:
⛏️ For consistency, we should make this an OffsetDateTime and pass it through the `redCapFormatDate` method like the CLI args